jstreegrid.js

```diff
@@ -41,11 +41,13 @@
     getIds = function (nodes) {
       return $.makeArray(nodes.map(function(){return this.id;}));
     },
-		findDataCell = function (uniq, ids, col) {
-			var ret = $(), columns = [].concat(col), cellId;
+    findDataCell = function (uniq, ids, col, scope) {
```

Comment:
OK, got it. I thought that was it, but wasn't sure. So `scope` limits the scope in which to search. If not provided, just use the document's DOM. Makes sense.